packages/mobile/src/components/audio/AudioPlayer.tsx

```diff
@@ -609,6 +610,8 @@ export const AudioPlayer = () => {
       return {
         url,
         type: TrackType.Default,
+        contentType: 'audio/mpeg',
+        pitchAlgorithm: PitchAlgorithm.Music,
```

Comment:
fun change, does this change audio quality?

Reply:
nah i don't think it actually does anything but my hope is that being more explicit would help the OS do something more correct. seems not bad to have though